### PR TITLE
Possible Fix for Modules Migration Error

### PR DESCRIPTION
**Pull request by YashGovekar**

### CODE REVIEW

File: app/Services/ModuleService.php [hunk 1]
```diff
@@ -240,6 +240,9 @@ public function updateModule($id, $status): bool
         $module->update([
             'enabled' => $status,
         ]);
+        if ($status === 1) {
```
Comment (nabeelio) [inline app/Services/ModuleService.php:243]: check for `true`

**Comment by nabeelio** [comment]
Were you able to duplicate the problem and then this fixed it?

File: app/Services/ModuleService.php (hunk 1 above)
Comment (nabeelio) [reply app/Services/ModuleService.php:243]: And add a newline above this one, hard to read without the spacing

**Comment by YashGovekar** [comment]
Actually I can't because I dont have any module that can migrate.

File: app/Services/ModuleService.php (hunk 1 above)
Comment (YashGovekar) [reply app/Services/ModuleService.php:243]: Alright

**Comment by YashGovekar** [comment]
This is really strange, I am not able to access the phpvms extracted from .tar.gz

**Comment by YashGovekar** [comment]
Fixed them!
Actually the module folder was not being called while doing `runAllMigrations()` in `MigrationService`
Hence, it was going in a loop.

**Comment by nabeelio** [comment]
Closing in favor of #883

File: app/Services/Installer/MigrationService.php [hunk 2]
```diff
@@ -68,6 +66,15 @@ public function runAllMigrations()
...
         $output .= trim(Artisan::output());
 
+        $modules = Module::allEnabled();
+        foreach ($modules as $module) {
+            $module_path = $module->getPath().'/Database/migrations';
+            if (file_exists($module_path)) {
+                Artisan::call('module:migrate '.$module->getName());
+                $output .= trim(Artisan::output());
+            }
+        }
```
Comment (nabeelio) [inline app/Services/Installer/MigrationService.php:76]: Ok, we kinda ended up doing the same thing :) Nice

Comment (YashGovekar) [reply app/Services/Installer/MigrationService.php:76]: Yeah, thanks :D

**Pull request closed without merging**